### ISSUE TITLE
[Frio] Back to top button blocks input fields

### ISSUE THREAD

**Issue by vinzv** [friendica/friendica#7560]
There are [complaints](https://less.re/display/e18176ef-325d-60f1-666e-133355463051) that the grey "back to top" arrow button blocks parts of the input field when replying. I can reproduce that as to be seen here:

![Screenshot_20190824_151212](https://user-images.githubusercontent.com/74432/63637897-3fcda280-c682-11e9-8168-261cf1ec5c44.png) ![Screenshot_20190824_151639](https://user-images.githubusercontent.com/74432/63637898-40663900-c682-11e9-8883-eb41380517e1.png)

To fix that we could either
- remove the button completely (my preference, but people might miss it)
or
- try to move it around (might be not really satisfying and beautiful)
or
- try to fix it the way that it's only shown when scrolling upwards (expect work to be done!)

So, what do you think?

**Comment by AlfredSK** [comment]
What's even more challenging, it sometimes blocks the submit button. 
I think the third option is the best one w/o losing the up-button completely.

I vote 'only shown when scrolling upwards' ✅

**Comment by vinzv** [comment]
This is strange: the function for fading the button out and back in on scroll is already there but somehow does not work reliable as it's depending on one fixed position. 

Which brought me to another solution: why not hide the button as soon as a textarea gets focus? To make it clear: as soon as you tap into a comment field, the button gets hidden. If you tap outside or scroll it's back.

If you want to test it yourself add the following lines to `view/theme/frio/js/theme.js` at line 22:

```javascript
        $('textarea').focus(function() {
                $('#back-to-top').fadeOut(0);
        });

        $('textarea').focusout(function() {
                $('#back-to-top').fadeIn(0);
        });
```

**Comment by AlfredSK** [comment]
I think this is fixed. Can we close it?

**Comment by MrPetovan** [comment]
Closed by #8271
